micro-reactive/build.gradle

```diff
@@ -2,7 +2,7 @@ description = 'micro-reactive'
 dependencies {
 	
 
-	compile 'com.aol.cyclops:cyclops-reactor:'+cyclopsVersion
+	//compile 'com.aol.cyclops:cyclops-reactor:'+cyclopsVersion
```

Comment:
should this import be removed instead of just commented out ?

Reply:
yes